assets/airports/kcvg.json

```diff
@@ -0,0 +1,6771 @@
+                ["ROCKT", "CADRE", "AHTIY*"]
+            ]
+        },
+        "SILKS4": {
+            "icao": "SILKS4",
+            "name": "Silks Four",
+            "rwy": {
+                "18C": [["FLOEE", "A14+"], "FEBLU"],
+                "18L": ["CAMUM", "ZEGUL"],
+                "18R": ["FANSA"],
+                "27": [["BONLE", "A14+"], "CULED"],
+                "36C": ["HOBAL", "BOUMR", "CULED"],
+                "36L": ["HOBAL", "BOUMR", "CULED"],
+                "36R": [["HAGSS", "A14+"], "EVAME", "CEMLA", "FRTZY", "CULED"]
+            },
+            "body": ["SILKS"],
+            "exitPoints": {
+                "TRFWA": ["TRFWA"]
+            },
+            "draw": [
+                ["FLOEE", "FEBLU", "SILKS"],
+                ["CAMUM", "ZEGUL", "SILKS"],
+                ["FANSA", "SILKS"],
+                ["BONLE", "CULED"],
+                ["HOBAL", "BOUMR", "CULED"],
+                ["HAGSS", "EVAME", "CEMLA", "FRTZY", "CULED", "SILKS", "TRFWA*"]
+            ]
+        }
+    },
+    "stars": {
+        "CEGRM4": {
+            "icao": "CEGRM4",
+            "name": "Cegram Four",
+            "entryPoints": {
+                "JOT": ["JOT", ["CADIZ", "A290"], ["FATOX", "A240"]]
+            },
+            "body": [["BVT", "A240"], ["TORIO", "A240"], ["MACES", "A190"], ["VHP", "A140"], ["SHB", "A140"], ["CEGRM", "A110"], "BCATT"],
+            "rwy": {
+                "09": ["BAMGE"],
+                "18C": ["BAMGE"],
+                "18L": ["BAMGE"],
+                "18R": ["BAMGE"],
+                "27": ["BAMGE"],
+                "36C": ["BAMGE", "FIDEK", "ZETIP"],
+                "36L": ["BAMGE", "FIDEK", "ZETIP"],
+                "36R": ["BAMGE", "FIDEK", "ZETIP"]
+            },
+            "draw": ["JOT", "CADIZ", "FATOX", "BVT", "TORIO", "MACES", "VHP", "SHB", "CEGRM", "BCATT", "BAMGE", "FIDEK", "ZETIP"]
+        },
+        "GAVNN4": {
+            "icao": "GAVNN4",
+            "name": "Gavin Four",
+            "entryPoints": {
+                "HNN": ["HNN"]
+            },
+            "body": [["JAYKB", "A240"], ["BRUSH", "A100"], ["GAVNN", "A100"], ["KNEDY", "A100"]],
+            "rwy": {
+                "09": ["AAJAY"],
+                "18C": ["AAJAY", "BACAS", "OGIRE"],
+                "18L": ["AAJAY", "BACAS", "OGIRE"],
+                "18R": ["AAJAY", "BACAS", "OGIRE"],
+                "27": ["AAJAY"],
+                "36C": ["AAJAY"],
+                "36L": ["AAJAY"],
+                "36R": ["AAJAY"]
+            },
+            "draw": ["HNN", "JAYKB", "BRUSH", "GAVNN", "KNEDY", "AAJAY", "BACAS", "OGIRE"]
+        },
+        "JAKIE4": {
+            "icao": "JAKIE4",
+            "name": "Jakie Four",
+            "entryPoints": {
+                "VXV": ["VXV"]
+            },
+            "body": [["ARTUR", "A190"], ["CHEDA", "A100"], ["ODABE", "A100"], ["HAYZL", "A100"], ["JAKIE", "A100"]],
+            "rwy": {
+                "09":  ["FLM"],
```

Comment:
"Landing NORTH, EAST, WEST: Expect RADAR vectors to final approach course prior to FIDEK"
--> rwy 09, 27, 36L/C/R should go up to FIDEK